rpm/cloud-hypervisor.spec

```diff
@@ -0,0 +1,94 @@
+Name:           cloud-hypervisor
+Summary:        Cloud Hypervisor is an open source Virtual Machine Monitor (VMM) that runs on top of KVM.
+Version:        0.8.0
+Release:        0%{?dist}
+License:        ASL 2.0 or BSD-3-clause
+Group:          Applications/System
+Vendor:         Microsoft Corporation
```

Comment:
Not sure this should be `Microsoft Corporation` here.

Reply:
Yes. I can remove this field as this is optional.